examples/rsp-cra-18/src/BodyContent.tsx

```diff
@@ -0,0 +1,26 @@
+import {Item, TabList, TabPanels, Tabs} from '@adobe/react-spectrum'
+import TodoList from './TodoList';
+import JournalList from './JournalList';
+
+function BodyContent(){
+    return(
+
+      <Tabs aria-label="Different Productivity Tabs">
+      <TabList>
```

Comment:
indentation is off for this file, we may want to turn on eslint to run in this directory
https://github.com/adobe/react-spectrum/blob/main/.eslintrc.js#L18